Code/GraphMol/ChemTransforms/MolFragmenter.h

```diff
@@ -100,5 +100,28 @@ RDKIT_CHEMTRANSFORMS_EXPORT void constructFragmenterBondTypes(
 RDKIT_CHEMTRANSFORMS_EXPORT void constructBRICSBondTypes(
     std::vector<FragmenterBondType> &defs);
 }  // namespace MolFragmenter
+
+
+enum class RDKIT_CHEMTRANSFORMS_EXPORT  MolzipLabel {
+    AtomMapNumber,
+    Isotope,
+    FragmentOnBonds,
+    AtomType };
+
+struct RDKIT_CHEMTRANSFORMS_EXPORT MolzipParams {
+  MolzipLabel label = MolzipLabel::AtomMapNumber;
+  std::vector<std::string> atomSymbols = {};
```

Comment:
I don't think the empty initializer is necessary here. `std::vector` default inializes to be empty